src/Microsoft.ML.Tokenizers/Model/TiktokenTokenizer.cs

```diff
@@ -1185,13 +1185,13 @@ private static (Dictionary<string, int> SpecialTokens, Regex Regex, string Vocab
         internal static partial Regex O200kBaseRegex();
 #else
         private static Regex? _cl100kBaseRegex;
-        private static Regex Cl100kBaseRegex() => _cl100kBaseRegex ??= new Regex(Cl100kBaseRegexPattern, RegexOptions.Compiled);
+        private static Regex Cl100kBaseRegex() => _cl100kBaseRegex ??= new Regex(Cl100kBaseRegexPattern, RegexOptions.Compiled, TimeSpan.FromMilliseconds(PreTokenizer.DefaultTimeOutInMilliseconds));
```

Comment:
Don't you also need to update the corresponding [GeneratedRegex(...)]?

Reply:
Yes, I missed that. Thanks for the catch.